app/controllers/ContentController.java

```diff
@@ -22,7 +25,7 @@
     private ThreadNotification sortNotification;
 
     @Inject
-    public ContentController(AbstractDataAccess access, ApplicationLifecycle appLifecycle) {
+    public ContentController(AbstractDataAccess access, ApplicationLifecycle appLifecycle) throws NetworkException, OAuthException {
```

Comment:
This class shouldn't be throwing implementation specific exceptions, they should be handled down in RedditSource()